_locales/en/messages.json

```diff
@@ -683,7 +683,7 @@
         "message": "Disarm motors after set delay [seconds] (Requires MOTOR_STOP feature)"
     },
     "configurationDisarmKillSwitch": {
-        "message": "Disarm motors regardless of throttle value (When arming via AUX channel)"
+        "message": "Arm/Disarm motors regardless of throttle value when using AUX channel"
```

Comment:
this could confuse a noob user like me, i'd prefer to read
`Disarm motors regardless of throttle value (when disarming via AUX channel)`
because having 'arming' in that sentence could be thought as : i can arm with throttle up (quite dangerous indeed)